bitnami/mongodb/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: mongodb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
-version: 14.2.5
+version: 14.2.6
```

I think is better to use a minor update because we are adding a new feature.
```suggestion
version: 14.3.0
```